src/scroll-timeline-css-parser.js

```diff
@@ -354,6 +375,48 @@ export class StyleParser {
     }
   }
 
+  extractScrollTimelineNames(contents) {
+    const value = RegexMatcher.ANIMATION_TIMELINE.exec(contents)[1].trim();
+    const timelineNames = [];
+
+    value.split(",").map(part => part.trim()).forEach(part => {
+      if(isAnonymousScrollTimeline(part)) {
+        const name = this.saveAnonymousTimelineName(part);
+        timelineNames.push(name);
```

Comment:
Possible for there to be a name collision between the auto-generated name for an anonymous scroll timeline and a named timeline.

Reply:
That's true, I added ":" to the beginning of anonymous timeline names to get rid of the collision situation. 
':' is not allowed to be used in identifiers, even after obfuscation.
animation-timeline: foo, scroll(), :bar;     
// ":bar" is not valid here 

I've stashed adding ":true", ":false" to the end of names, please let me know if adding ':' to the beginning of anonymous timeline names doesn't work, then I'll upload the other approach.

Reply:
: prefix looks good for disambiguating.